src/components/video/index.tsx

```diff
@@ -34,6 +34,7 @@ export type Props = {
   inputProps?: React.InputHTMLAttributes<HTMLInputElement>;
   onSave?: () => void;
   onError?: (error: Error) => void;
+  errorComponent?: React.ComponentType<any>;
```

Comment:
Since the changes here are practically equivalent to the image one, the same comments apply :)

```suggestion
  errorComponent?: React.ComponentType<{ error: Error }>;
```